content/src/cloud/features/cloud-backup-enable.md

```diff
@@ -6,3 +6,30 @@ zendesk:
   labels: cloud
 ---
 
+If your backup is smaller than 5GB in size, you have the option to store an encrypted backup on Home Assistant Cloud.
+
+## Prerequisites
+
+A backup will only be uploaded to Home Assistant Cloud if the following requirements are met:
+
+- You are running Home Assistant 2025.1 or later
+- Automatic backups have been enabled
+- The backup is under 5GB in size
+
+## To setup a Home Assistant Cloud backup
+
+**_Note_**: Home Assistant Cloud will only store one backup at a time on Home Assistant Cloud. It will always store your most recently made encrypted backup.
+
+<div class='videoWrapper'>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/nSSkMVBqChM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
```

Comment:
This should probably be an include.

`{% include youtube https://www.youtube.com/embed/nSSkMVBqChM %}` or something.